pyseir/inference/initial_conditions_fitter.py

```diff
@@ -12,11 +12,13 @@
 from multiprocessing import Pool
 from functools import partial
 import seaborn as sns
+from libs.datasets import combined_datasets
+from libs.datasets import CommonFields
 from pyseir import load_data
 from pyseir import OUTPUT_DIR
 
 
-class InitialConditionsFitter:
+class InitialConditionsFitter(object):
```

Comment:
I don't think this does anything https://stackoverflow.com/questions/1238606/is-it-necessary-or-useful-to-inherit-from-pythons-object-in-python-3-x

Reply:
Ah you're right - I generally have followed the google style guid pretty strictly, but it's probably a bit outdated in a purely python 3 world